app/src/main/java/phone/vishnu/dialogmusicplayer/MainActivity.java

```diff
@@ -272,7 +275,7 @@ public void disableScreenRotation() {
         if (orientation == Configuration.ORIENTATION_LANDSCAPE)
             setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_SENSOR_LANDSCAPE);
         else if (orientation == Configuration.ORIENTATION_PORTRAIT)
-            setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
+            setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_UNSPECIFIED);
```

Comment:
Why? This would cause the same music to play multiple times. This was a hack to avoid that!